repo/packages/B/beta-kubernetes/8/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "5.0",
+  "minDcosReleaseVersion": "1.12",
+  "upgradesFrom": [
+    "*"
+  ],
+  "name": "beta-kubernetes",
+  "version": "9.9.9-9.9.9-beta",
```

Comment:
I am also wondering how we can release subsequent betas.

Reply:
Does it make sense to have the version be `2.0.0-1.12.0-beta.0`?